Pod/Classes/GDGColumn.m

```diff
@@ -58,4 +57,16 @@ - (NSString *)fullName
 	return [self.source.alias stringByAppendingFormat:@".%@", _name];
 }
 
+- (GDGColumn *)copyWithZone:(nullable NSZone *)zone
+{
+	GDGColumn *copy = [(GDGColumn *) [[self class] alloc] init];
```

Comment:
> You are being more verbose that if you just write [[GDGColumn allocWithZone:zone] init]
It has to be like that. If someone inherits from `GDGColumn`, the allocted object must be `[self class]`, not `GDGColumn`.